[INFO   ] Logger: Record log in /storage/emulated/0/code/util/DownWithYT/v1.1/.kivy/logs/kivy_22-10-04_10.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  9 2021, 08:02:40) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 build 1.9@4893595'>
[INFO   ] GL: OpenGL vendor <b'Imagination Technologies'>
[INFO   ] GL: OpenGL renderer <b'PowerVR Rogue GE8300'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <4096>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Text: Provider: sdl2
[INFO   ] Start: Started App
[WARNING] Base: Unknown <android> provider
[INFO   ] Base: Start application main loop
[INFO   ] GL: NPOT texture support is available
[INFO   ] USE: YT_Widget
[INFO   ] USE: YT_Widget
[INFO   ] USE: YT_Widget
[INFO   ] USE: YT_Widget
[INFO   ] USE: YT_Widget
[INFO   ] USE: YT_Widget
[INFO   ] USE: YT_Widget
[INFO   ] USE: YT_Widget
[INFO   ] USE: YT_Widget
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/code/util/DownWithYT/v1.1/dwytapp.py", line 103, in <module>
[WARNING] stderr:     scr.run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/app.py", line 950, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 582, in runTouchApp
[WARNING] stderr:     EventLoop.mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 347, in mainloop
[WARNING] stderr:     self.idle()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 399, in idle
[WARNING] stderr:     Clock.tick_draw()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/clock.py", line 783, in tick_draw
[WARNING] stderr:     self._process_events_before_frame()
[WARNING] stderr:   File "kivy/_clock.pyx", line 658, in kivy._clock.CyClockBase._process_events_before_frame
[WARNING] stderr:   File "kivy/_clock.pyx", line 701, in kivy._clock.CyClockBase._process_events_before_frame
[WARNING] stderr:   File "kivy/_clock.pyx", line 697, in kivy._clock.CyClockBase._process_events_before_frame
[WARNING] stderr:   File "kivy/_clock.pyx", line 218, in kivy._clock.ClockEvent.tick
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/uix/boxlayout.py", line 310, in do_layout
[WARNING] stderr:     for i, x, y, w, h in self._iterate_layout(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/uix/boxlayout.py", line 267, in _iterate_layout
[WARNING] stderr:     for key, value in pos_hint.items():
[WARNING] stderr: AttributeError: 'set' object has no attribute 'items'
